src/hooks/useGeneratedKey.ts

```diff
@@ -0,0 +1,37 @@
+//Keep track of all the keys we've generated to ensure uniqueness
+let usedKeys: Array<string> = [];
+
+//Generates a random key
+const keyGenerator = (): string => {
+  return `portalize_${Math.random().toString(36).substr(2, 16)}-${Math.random()
+    .toString(36)
+    .substr(2, 16)}-${Math.random().toString(36).substr(2, 16)}`;
+};
+
+//Custom hook that checks for uniqueness and retries if clashes
```

Comment:
Can you add a space after the comment char

```suggestion
// Custom hook that checks for uniqueness and retries if clashes
```